mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.kt

```diff
@@ -331,55 +331,53 @@ class MockWebServer : ExternalResource(), Closeable {
     serverSocket!!.bind(inetSocketAddress, 50)
 
     port = serverSocket!!.localPort
-    executor!!.execute(object : NamedRunnable("MockWebServer $port") {
-      override fun execute() {
-        try {
-          logger.info("${this@MockWebServer} starting to accept connections")
-          acceptConnections()
-        } catch (e: Throwable) {
-          logger.log(Level.WARNING, "${this@MockWebServer}  failed unexpectedly", e)
-        }
+    executor!!.execute("MockWebServer $port") {
```

Comment:
GitHub doesn’t do this diff well. It’s a 17-line diff when you ignore whitespace.

Reply:
https://github.com/square/okhttp/pull/5038/files?w=1